### DISCUSSION TITLE
What does the value "maybetemp" from an Apollo oil level sensor mean?

### DISCUSSION THREAD

**Discussion by rowanbradley** [merbanan/rtl_433#2853] [Q&A]
I'm using rtl_433 to read the values from my Apollo oil level meter. It returns two apparently temperature related values, temperature_C and maybetemp. They both have very different values, e.g. maybetemp = 21, temperature_C = 13.333. Which of these, or which combination of them, represents the actual temperature of the sensor (and presumably the air between the sensor and the oil surface, and possibly the oil itself)?

Thank you - Rowan

**Comment by zuckschwerdt** [comment]
The original author, David Woodhouse, took a guess. Somebody still has to verify or improve that.

https://github.com/merbanan/rtl_433/blob/152a72f93ae5a965553745157daf2a2e0dffdb67/src/devices/oil_watchman.c#L62-L65
